src/main/java/TaskList.java

```diff
@@ -0,0 +1,27 @@
+public class TaskList {
+    private final Task[] Tasks = new Task[100];
+    private int totalTasks = 0;
+    private final String solidLine = "\n------------------------------------------------------------------------------------------------------------------------------\n";
```

Comment:
Constant name should be in caps.
```suggestion
    private final String SOLIDLINE= "\n------------------------------------------------------------------------------------------------------------------------------\n";
```